mobile/lib/features/trade/trade_bottom_sheet_confirmation.dart

```diff
@@ -177,10 +177,10 @@ class TradeBottomSheetConfirmation extends StatelessWidget {
     TextStyle dataRowStyle = const TextStyle(fontSize: 14);
 
     final orderMatchingFee = tradeValues.fee ?? Amount.zero();
-    final feeRebate = referralStatus != null
-        ? Amount((referralStatus.referralFeeBonus * orderMatchingFee.sats).ceil())
+    final feeBeforeRebate = referralStatus != null && orderMatchingFee.sats > 0
+        ? Amount((orderMatchingFee.sats / (1.0 - referralStatus.referralFeeBonus)).floor())
```

Comment:
🤔 I think this can lead to a division-by-zero error.

Reply:
If we have a fee rebate of 100%. I'll add a handling

Reply:
Yeah. I only mention it because we talked about this scenario earlier today.

Reply:
Hmm. But if we have a fee rebate of 100% the order matching fee will be 0.